fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -163,6 +169,10 @@ public TableFunctionTable(Map<String, String> properties) throws DdlException {
         columns.addAll(getSchemaFromPath());
```

Comment:
Is this ok to put such a network call and reading file operation in a object constructor?

Reply:
it's ok